pkgs/development/python-modules/imageio/default.nix

```diff
@@ -1,28 +1,34 @@
 { stdenv
 , buildPythonPackage
+, pathlib
 , fetchPypi
 , pillow
 , psutil
+, imageio-ffmpeg
 , pytest
 , numpy
 , isPy3k
+, ffmpeg
 , futures
 , enum34
 }:
 
 buildPythonPackage rec {
   pname = "imageio";
-  version = "2.4.1";
+  version = "2.5.0";
 
   src = fetchPypi {
-    sha256 = "0jjiwf6wjipmykh33prjh448qv8mpgngfi77ndc7mym5r1xhgf0n";
+    sha256 = "1bdcrr5190jvk0msw2lswj4pbdhrcggjpj8m6q2a2mrxzjnmmrj2";
     inherit pname version;
   };
 
-  checkInputs = [ pytest psutil ];
+  checkInputs = [ pytest psutil ] ++ stdenv.lib.optionals isPy3k [
+    imageio-ffmpeg ffmpeg
+    ] ++ stdenv.lib.optionals (!isPy3k) [ pathlib ];
```

Comment:
@pmiddend `pathlib` is not an optional dependency and should thus be moved to `propagatedBuildInputs`.

Reply:
Oh, I see it's already there. No need to have it in two places.